[2025-10-10T10:19:43.696+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: inventory_analytics_pipeline.visualize_inventory scheduled__2025-10-09T00:00:00+00:00 [queued]>
[2025-10-10T10:19:44.223+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: inventory_analytics_pipeline.visualize_inventory scheduled__2025-10-09T00:00:00+00:00 [queued]>
[2025-10-10T10:19:44.286+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-10-10T10:19:44.826+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): visualize_inventory> on 2025-10-09 00:00:00+00:00
[2025-10-10T10:19:44.978+0000] {standard_task_runner.py:60} INFO - Started process 209 to run task
[2025-10-10T10:19:45.147+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'inventory_analytics_pipeline', 'visualize_inventory', 'scheduled__2025-10-09T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/inventory_analytics_pipeline.py', '--cfg-path', '/tmp/tmpz3gopp8g']
[2025-10-10T10:19:45.310+0000] {standard_task_runner.py:88} INFO - Job 30: Subtask visualize_inventory
[2025-10-10T10:19:45.803+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-10T10:19:46.732+0000] {task_command.py:423} INFO - Running <TaskInstance: inventory_analytics_pipeline.visualize_inventory scheduled__2025-10-09T00:00:00+00:00 [running]> on host ea4788e59d79
[2025-10-10T10:19:49.284+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='inventory_analytics_pipeline' AIRFLOW_CTX_TASK_ID='visualize_inventory' AIRFLOW_CTX_EXECUTION_DATE='2025-10-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-09T00:00:00+00:00'
[2025-10-10T10:20:06.288+0000] {logging_mixin.py:188} INFO - âœ… Inventory plot saved: /opt/***/data/plots/inventory_plot_20251010_101958.png
[2025-10-10T10:20:06.334+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-10-10T10:20:06.474+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=inventory_analytics_pipeline, task_id=visualize_inventory, execution_date=20251009T000000, start_date=20251010T101943, end_date=20251010T102006
[2025-10-10T10:20:07.087+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-10T10:20:07.545+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
